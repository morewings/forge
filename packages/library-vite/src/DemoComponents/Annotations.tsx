import React, { FC } from 'react';

export type AnnotationsProps = {
    /** Example annotation */
    title: string;
};

export const Annotations: FC<AnnotationsProps> = ({ title }) => {
    return <div>{title}</div>;
};
