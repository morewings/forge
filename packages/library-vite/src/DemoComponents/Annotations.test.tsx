import { render } from '@testing-library/react';
import { Annotations } from './Annotations';

describe('Annotations', () => {
    it('renders', () => {
        const { asFragment, getByText } = render(<Annotations title="foo" />);
        expect(asFragment()).toMatchSnapshot();
        expect(getByText('foo')).toBeInTheDocument();
    });
});
