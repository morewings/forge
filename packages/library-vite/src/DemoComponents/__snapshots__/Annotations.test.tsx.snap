// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Annotations renders 1`] = `
<DocumentFragment>
  <div>
    foo
  </div>
</DocumentFragment>
`;
